examples/cpu_temperature.py

```diff
@@ -0,0 +1,60 @@
+import colorsys
+import time
+
+import openrazer.client
+
+# These settings are nice if your CPU temperature is a bit less than 30°C
+# at idle and a bit more than 60°C at full load, tune them for your needs
+tGreen = 30
+tRed = 60
+
+tRange = 3 * (tRed - tGreen)
+colour0 = tRed / tRange
+tRange = 1000 * tRange
+
+devman = openrazer.client.DeviceManager()
+
+# Show temperature colour on this device
+device = None
+for d in devman.devices:
+    if "Razer BlackWidow Chroma" in d.name:
+        device = d
```

Comment:
Here we only take BlackWidow but further down you write about Mamba TE? I'd also be find if the example effect just takes the first device (or all) or you give it an index integer as parameter basically, or device name or something.

Reply:
I've changed it to apply to any connected device(s).  I was initially thinking it's just an example to copy/paste/study on how to select a device, rather then a mini program in itself. But, it should be usable to simply run for _any_ device now.

Also, I've changed the code so it can be the first example to demonstrate the endpoints for a device's different zones. The other examples use advanced/matrix or `sync_effects = True` . Originally put that in due to that upcoming API ~_breakage_~ change where some devices moved from `.fx` to `.fx.misc.backlight` like the Mamba TE.

Reply:
super nice, thanks!